<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>215</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Export Type</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,2,0">
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Choose Export</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Export Methods</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rad_sra">
          <property name="text">
           <string>Export SRA Links</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rad_sra_assembly">
          <property name="text">
           <string>Export complementary SRA and Assembly Links</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rad_bactopia">
          <property name="text">
           <string>Export Bactopia FOFN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rad_asa3p">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Export ASA3P</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_types">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Export Types</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="rad_ncbi">
          <property name="text">
           <string>NCBI</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rad_aws">
          <property name="text">
           <string>AWS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rad_gcp">
          <property name="text">
           <string>GCP</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="but_cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="but_export">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rad_bactopia</sender>
   <signal>toggled(bool)</signal>
   <receiver>gb_types</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rad_asa3p</sender>
   <signal>toggled(bool)</signal>
   <receiver>gb_types</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
